### ISSUE TITLE
Find in files to search specific extensions. 

### ISSUE THREAD

**Issue by trideepgogoi** [adobe/brackets#12005]
Sometimes there is a need to find all references in files of a specific extension. (for example search for something in ALL *.js files in a project OR *.json files)

There should be support to search in all files with a specified extension.

**Comment by abose** [comment]
This is a good feature to have, Will try to implement this for 1.6. 
Maybe `inclusion sets` like `exclusion sets` in file filters?

**Comment by abose** [comment]
Or a fuzzy search like something in google
`"search string" in:"*.js,*.css"`

**Comment by abose** [comment]
@petetnt @swmitra Any ideas on this workflow?

**Comment by petetnt** [comment]
@abose I think the easiest (best) way to achieve this would be using the globbing patterns from [node-glob](https://github.com/isaacs/node-glob). 

You could have a filter (instead of exclusion and inclusion sets), that would default to `**`. If you wanted to search everything but `.js` files, you could apply `!**/*.js`, or to filter `.json` files only you'd only have to apply `**/*.json`. Many developers are already familiar to the globbing sets and the current [File Filter](https://github.com/adobe/brackets/wiki/Using-File-Filters) syntax is pretty much a subset of the `node-glob` patterns anyway, as seen in the [current implementation](https://github.com/adobe/brackets/blob/2a832d0059c9b5b67fb868a8672764fe728ac7f7/src/search/FileFilters.js#L206) where the patterns are already called `glob`s.

The drawback would be that existing `Exclusion sets` would have to be converted to the new format, *if* they are not compatible as-is.

**Comment by abose** [comment]
Yeah, we could rename as filter sets and put in an in browser version of node-glob in. 
Will check it out
Also will be usefull to put in a fuzzy mode like `"search string" in:"*.js,*.css"` or `"search string" notin:"*.js,*.css,node_modules"` for adhoc quick filtration.

**Comment by trideepgogoi** [comment]
@petetnt node-glob makes a lot of sense to an end user like me. As mentioned we already use globs and most are familiar with it.
